tools/automation/verify/verify_module_load_times.py

```diff
@@ -11,11 +11,17 @@
 import sys
 
 
+TOTAL = 'ALL'
 NUM_RUNS = 3
-
-HIGH_THRESHOLD = 35
-HIGH_MODULES = ['network', 'vm', 'batch', 'storage']
-LO_THRESHOLD = 10
+DEFAULT_THRESHOLD = 10
+# explicit thresholds that deviate from the default
+THRESHOLDS = {
+    'network': 25,
+    'vm': 25,
+    'batch': 25,
+    'storage': 50,
+    TOTAL: 300
+}
```

Comment:
It would be nice if these values can be configured through environment variable.

Reply:
If these values end up being problematic, then I'd have to use a more sophisticated method to essentially determine a scalable load-time profile and detect deviations from that.  However, since we are always running these on a stable build, my experience (and hope!) is that we won't see wild variations.